### DISCUSSION TITLE
Stop SolarEdge battery discharging to charge EV

### DISCUSSION THREAD

**Discussion by buttash** [springfall2008/batpred#1593] [Q&A]
I have Octopus Intelligent configured to control when my EV is charged.  The car charges but draws from the my battery.  I'm not sure if I've misconfigured my set-up or if I need to detect charging and set the battery mode to not discharge outside of Predbat.  Can anyone shed some light on this?
Thanks

**Comment by gcoan** [comment]
Have you configured Predbat to be aware of the Octopus intelligent charging - see https://springfall2008.github.io/batpred/car-charging/ ?

Predbat should automatically set the battery to charge or at least pause when Octopus intelligent is charging your car to stop the battery draining to the EV

**Comment by buttash** [comment]
Thanks for getting back on this.  I have configured Octopus Intelligent - but on revisiting the apps.yaml I found that I had re:xxx on a couple of Octopus parameters and not re:(xxxx).  This meant that my account was being pulled through to octopus_intelligent_slot and octopus_ready_time.  I have fixed this and reloaded.  Hopefully it'll solve this issue for me.

**Reply by gcoan** [reply]
Was that a typo on your configuration of apps.yaml or an error in the standard predbat template (that needs fixing)?

Good luck, sounds positive though

**Reply by buttash** [reply]
The standard template is formatted correctly.
